homeassistant/components/myuplink/strings.json

```diff
@@ -41,5 +41,25 @@
         "name": "Status"
       }
     }
+  },
+  "exceptions": {
+    "config_entry_auth_failed": {
+      "message": "Error while logging in to the API."
```

Comment:
I think we could make it clearer to the user that this was due to invalid credentials

Reply:
Well, there can be other reasons too. Invalid credentials is a common reason though.